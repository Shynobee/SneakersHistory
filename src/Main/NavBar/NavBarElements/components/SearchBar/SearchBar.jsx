import React from 'react';
import PropTypes from 'prop-types';
import { link } from 'react-router';
import {
  withStyles,
  TextField
} from '@material-ui/core';

const styles = withStyles({
  root: {
    
  },
  textField: {

  }
});

class SearchBar extends React.Component {

  render() {
    const { classes } = this.props;
    return (
      <div className={classes.root}>
        <TextField
          id="outlined-search"
          label="Search field"
          type="search"
          className={classes.textField}
          margin="normal"
          variant="outlined"
        />
      </div>
    );
  }
}

SearchBar.propTypes = {
  classes: PropTypes.object.isRequired,
};

const SearchBarWithStyle = withStyles(styles)(SearchBar);

export default SearchBarWithStyle;