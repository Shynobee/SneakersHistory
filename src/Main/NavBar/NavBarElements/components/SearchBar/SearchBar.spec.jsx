import React from 'react';
import ReactDOM from 'react-dom';
import SearchBar from './SearchBar';

describe('SearchBar', () => {
  it('Should renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<SearchBar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('Should match snapshot', () => {
    expect(SearchBar).toMatchSnapshot();
  });
});
