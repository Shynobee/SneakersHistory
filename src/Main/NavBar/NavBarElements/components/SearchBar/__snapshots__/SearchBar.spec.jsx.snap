// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Should match snapshot 1`] = `[Function]`;
